packages/react/src/components/Notification/Notification.mdx

```diff
@@ -65,6 +65,27 @@ an `aria-describedby` on the interactive element with a value that matches the
 
 <Canvas id="components-notifications-actionable--default" />
 
+### Using the `role` prop
+
+ActionableNotification's `role` prop allows you to provide a different
+`aria-role` than the default, `alertdialog`.
+
+This prop should be used with caution as it will disable the wrapping of focus
```

Comment:
minor possible wording update. also, noticed it in the prop description too
```suggestion
This prop should be used with caution as it will disable the focus wrapping
```